app/controlplane/plugins/core/dependency-track/v1/extension.go

```diff
@@ -143,67 +144,117 @@ func (i *DependencyTrack) Attach(ctx context.Context, req *sdk.AttachmentRequest
 	return &sdk.AttachmentResponse{Configuration: rawConfig}, nil
 }
 
-// Send the SBOM to the configured Dependency Track instance
+// Send the SBOMs to the configured Dependency Track instance
 func (i *DependencyTrack) Execute(ctx context.Context, req *sdk.ExecutionRequest) error {
-	i.Logger.Info("execution requested")
-
+	var errs error
 	// Iterate over all SBOMs
 	for _, sbom := range req.Input.Materials {
-		// Make sure it's an SBOM and all the required configuration has been received
-		if err := validateExecuteOpts(sbom, req.RegistrationInfo, req.AttachmentInfo); err != nil {
-			return fmt.Errorf("running validation: %w", err)
+		if err := doExecute(ctx, req, sbom, i.Logger); err != nil {
+			errs = errors.Join(errs, err)
```

Comment:
I don't know, what do you suggest? I thought that not failing is the safer bet, specially now that we have dynamic configuration. We can not guarantee rollbacks or ordering, so stopping the whole execution doesn't seem to give us much value, but I am curious to hear what you have in your mind.

Reply:
I prefer to fail quickly and loudly, but you are right. There is no transaction. If one DT SBOM execution succeeds, we will not be able to roll back anyway if the next one fails - at least not for now.

Reply:
> I prefer to fail quickly and loudly,

I agree. Note that if one of the SBOMs failed, the method will still return an error. In theory someday this will mean marking the integration as failed notifying the operator. What we don't do is to fail early but instead give a chance to all the SBOMs to be sent.